runtime-modules/content/src/lib.rs

```diff
@@ -2392,6 +2407,18 @@ decl_module! {
                 Self::validate_member_set(&params.transfer_params.new_collaborators.keys().cloned().collect())?;
             }
 
+            if let Ok(token_id) = channel.ensure_creator_token_issued::<T>() {
+                ensure!(
+                    T::ProjectToken::is_revenue_split_inactive(token_id),
+                    Error::<T>::ChannelTransfersBlockedDuringRevenueSplits
+                );
+
+                ensure!(
+                    T::ProjectToken::is_sale_unscheduled(token_id),
+                    Error::<T>::ChannelTransfersBlockedDuringTokenSales,
+                );
+            }
```

Comment:
Missing test coverage for those cases (ideally testing upcoming / ongoing / "finished but unfinalized" case for sales and the same for revenue splits)